src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -412,6 +448,7 @@ const filteredCode = computed(function() {
     })
     .join('\n')
 })
+const finalCode = computed(() => props.isSingleLine ? codeRef.value?.replace('\n', '') : props.code)
```

Comment:
I don't think you actually need the `codeRef` variable or `toRefs` up above where it's created since you're not mutating the value.

You're simply grabbing the prop value and replacing the spaces, so instead I think you can still just point to the prop value

```suggestion
const finalCode = computed(() => props.isSingleLine ? props.code.replace('\n', '') : props.code)
```

Reply:
Fixed